src/pages/index.js

```diff
@@ -81,45 +154,106 @@ const newCardPopup = new PopupWithForm({
 });
 newCardPopup.setEventListeners();
 
+const userInfo = new UserInfo({
+  profileTitleSelector: ".profile__title",
+  profileDescriptionSelector: ".profile__description",
+  avatarSelector: ".profile__image",
+});
+Promise.all([userInfo]);
```

Comment:
Needs correction: if you use Promise.all here you should include several promises, so you will receibe both user info and the cards together 

```js
Promise.all([api.getUserInfo(), api.getInitialCards()])
  .then(([userInfoResponse, cardsResponse]) => {..})
  .catch(..)
```